src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -853,6 +853,11 @@ def __call__(
             aspect_ratio = torch.tensor([float(height / width)]).repeat(batch_size * num_images_per_prompt, 1)
             resolution = resolution.to(dtype=prompt_embeds.dtype, device=device)
             aspect_ratio = aspect_ratio.to(dtype=prompt_embeds.dtype, device=device)
+
+            if do_classifier_free_guidance:
+                resolution = torch.cat([resolution, resolution], dim=0)
+                aspect_ratio = torch.cat([aspect_ratio, aspect_ratio], dim=0)
```

Comment:
Seems like a new addition?

Reply:
not really - it gets duplicated later inside `CombinedTimestepSizeEmbeddings`

https://github.com/huggingface/diffusers/blob/6976cab7caa959bff46896c52ec03d9d414fc689/src/diffusers/models/embeddings.py#L758

Reply:
Very nice <3

Reply:
@sayakpaul 
fast tests fail because there are some randomly initialized weights in some components. I think we need to put `torch.manual_seed(0)` before making each component e.g.
https://github.com/huggingface/diffusers/blob/6976cab7caa959bff46896c52ec03d9d414fc689/tests/pipelines/pixart/test_pixart.py#L67

should we open a new PR to only update the tests, and I rebase after that? I'm not comfortable updating tests directly from this PR since I updated the code

Reply:
But that wasn't the case before. Wonder what changed.